File: Code 3 Callouts/Code 3 Callouts/Models/Vehicles/IVehicle.cs
using Rage;
using Stealth.Common.Models;
using System.Drawing;
using static Stealth.Common.Natives.Vehicles;

namespace Stealth.Plugins.Code3Callouts.Models.Vehicles
{

	internal interface IVehicle
	{

		void Init();
		void FillColorValues();
		void CreateBlip(Color? color = null);

		void DeleteBlip();
		VehicleColor Colors { get; }
		EPaint PrimaryColorEnum { get; }
		string PrimaryColorName { get; }
		EPaint SecondaryColorEnum { get; }
		string SecondaryColorName { get; }
		Blip Blip { get; set; }

		string Name { get; set; }
	}

}

File: Code 3 Callouts/Code 3 Callouts/Models/Vehicles/IVehicle.vb
Imports Rage
Imports Stealth.Common.Models
Imports Stealth.Common.Natives.Vehicles

Namespace Models.Vehicles

    Public Interface IVehicle

        Sub Init()
        Sub FillColorValues()
        Sub CreateBlip(Optional ByVal color As Drawing.Color = Nothing)
        Sub DeleteBlip()

        ReadOnly Property Colors As VehicleColor
        ReadOnly Property PrimaryColorEnum As EPaint
        ReadOnly Property PrimaryColorName As String
        ReadOnly Property SecondaryColorEnum As EPaint
        ReadOnly Property SecondaryColorName As String
        Property Blip As Blip
        Property Name As String

    End Interface

End Namespace